core/src/main/java/org/mskcc/cbio/portal/util/ExtendedMutationUtil.java

```diff
@@ -47,319 +47,296 @@
 /**
  * Utility class related to ExtendedMutation.
  */
-public class ExtendedMutationUtil
-{
-	// originally NA (case ignored) was the only value for empty values in this column,
-	// but [Not Available] occurs in many TCGA studies.
-	// TODO if this list grows, create NotAvailableValues enum like in ImportClinicalData.java
-	public static final List <String> NOT_AVAILABLE = Arrays.asList("NA", "[Not Available]");
-
-	public static String getCaseId(String barCode)
-	{
-		// process bar code
-		// an example bar code looks like this:  TCGA-13-1479-01A-01W
-
-		String barCodeParts[] = barCode.split("-");
-
-		String caseId = null;
-
-		try
-		{
-			caseId = barCodeParts[0] + "-" + barCodeParts[1] + "-" + barCodeParts[2];
-
-			// the following condition was prompted by case ids coming from
-			// private cancer studies (like SKCM_BROAD) with case id's of
-			// the form MEL-JWCI-WGS-XX or MEL-Ma-Mel-XX or MEL-UKRV-Mel-XX
-			if (!barCode.startsWith("TCGA") &&
-			    barCodeParts.length == 4)
-			{
-				caseId += "-" + barCodeParts[3];
-			}
-		} catch (ArrayIndexOutOfBoundsException e) {
-			caseId = barCode;
-		}
-
-		return caseId;
-	}
-
-	/**
-	 * Determines the most accurate protein change value for the given mutation.
-	 *
-	 * If there is an annotator value, returns that value.
-	 * If no annotator value, then tries Amino Acid Change value.
-	 * If none of the above is valid then returns "MUTATED".
-	 *
-	 * @param parts     current mutation as split parts of the line
-	 * @param record    MAF record for the current line
-	 * @return          most accurate protein change
-	 */
-	public static String getProteinChange(String[] parts, MafRecord record)
-	{
-		// try annotator value first
-		//String proteinChange = record.getOncotatorProteinChange();
```

Comment:
removed commented out reference to `record.getOncotatorProteinChange()`

Reply:
this indicates that for this field at least, it was a conscious decision to move away from the oncotator value